src/OnboardingSPA/components/StartOptions/index.js

```diff
@@ -53,9 +55,9 @@ const StartOptions = ( { questionnaire, oldFlow, options } ) => {
 	};
 	const selectFlow = ( flow ) => {
 		switch ( flow ) {
-			case 'onboarding':
+			case ONBOARDING_FLOW:
```

Comment:
Should we maintain consistency by changing the terms onboarding, ai and sitegen to align with what we already have, which are, wp-setup, ecommerce, and sitegen.

Reply:
I was doubtful as of ONBOARDING_FLOW covers both wp-setup as well as ecommerce, lemme look and make it aligned